src/boostact/Boostact.js

```diff
@@ -161,7 +163,7 @@ const isUnchanged = (curChild, vChild) => {
 const determineState = (curChild, vChild) => {
   const sameType = curChild && vChild && curChild.type === vChild.type;
 
-  if (vChild?.parent.effectTag === "PLACEMENT") {
+  if (vChild && vChild.parent.effectTag === "PLACEMENT") {
```

Comment:
```suggestion
  if (vChild.parent.effectTag === "PLACEMENT") {
```
vChild가 무조건 존재하는것같아요

Reply:
그렇군요 PLACEMENT니깐.